## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 432888`
Commit: `2d57d53a8afa2142aea700bb191c0c0392872ca2`

---
### `aarch64-darwin`
<details>
  <summary>:x: 171 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.fnllm</li>
    <li>python312Packages.fnllm.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-cli</li>
    <li>python312Packages.langgraph-cli.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-runtime-inmem</li>
    <li>python312Packages.langgraph-runtime-inmem.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llm-sentence-transformers</li>
    <li>python312Packages.llm-sentence-transformers.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.smolagents</li>
    <li>python312Packages.smolagents.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.fnllm</li>
    <li>python313Packages.fnllm.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 371 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist</li>
    <li>alpaca</li>
    <li>chatblade</li>
    <li>chatblade.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>imaginer</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>litellm</li>
    <li>litellm.dist</li>
    <li>llm</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>openai (python313Packages.openai)</li>
    <li>openai.dist (python313Packages.openai.dist)</li>
    <li>python312Packages.banks</li>
    <li>python312Packages.banks.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.fireworks-ai</li>
    <li>python312Packages.fireworks-ai.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.llm</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.llm-cmd</li>
    <li>python312Packages.llm-cmd.dist</li>
    <li>python312Packages.llm-command-r</li>
    <li>python312Packages.llm-command-r.dist</li>
    <li>python312Packages.llm-deepseek</li>
    <li>python312Packages.llm-deepseek.dist</li>
    <li>python312Packages.llm-docs</li>
    <li>python312Packages.llm-docs.dist</li>
    <li>python312Packages.llm-echo</li>
    <li>python312Packages.llm-echo.dist</li>
    <li>python312Packages.llm-fragments-github</li>
    <li>python312Packages.llm-fragments-github.dist</li>
    <li>python312Packages.llm-fragments-pypi</li>
    <li>python312Packages.llm-fragments-pypi.dist</li>
    <li>python312Packages.llm-fragments-reader</li>
    <li>python312Packages.llm-fragments-reader.dist</li>
    <li>python312Packages.llm-fragments-symbex</li>
    <li>python312Packages.llm-fragments-symbex.dist</li>
    <li>python312Packages.llm-gemini</li>
    <li>python312Packages.llm-gemini.dist</li>
    <li>python312Packages.llm-gguf</li>
    <li>python312Packages.llm-gguf.dist</li>
    <li>python312Packages.llm-git</li>
    <li>python312Packages.llm-git.dist</li>
    <li>python312Packages.llm-github-copilot</li>
    <li>python312Packages.llm-github-copilot.dist</li>
    <li>python312Packages.llm-grok</li>
    <li>python312Packages.llm-grok.dist</li>
    <li>python312Packages.llm-groq</li>
    <li>python312Packages.llm-groq.dist</li>
    <li>python312Packages.llm-hacker-news</li>
    <li>python312Packages.llm-hacker-news.dist</li>
    <li>python312Packages.llm-jq</li>
    <li>python312Packages.llm-jq.dist</li>
    <li>python312Packages.llm-llama-server</li>
    <li>python312Packages.llm-llama-server.dist</li>
    <li>python312Packages.llm-mistral</li>
    <li>python312Packages.llm-mistral.dist</li>
    <li>python312Packages.llm-ollama</li>
    <li>python312Packages.llm-ollama.dist</li>
    <li>python312Packages.llm-openai-plugin</li>
    <li>python312Packages.llm-openai-plugin.dist</li>
    <li>python312Packages.llm-openrouter</li>
    <li>python312Packages.llm-openrouter.dist</li>
    <li>python312Packages.llm-pdf-to-images</li>
    <li>python312Packages.llm-pdf-to-images.dist</li>
    <li>python312Packages.llm-perplexity</li>
    <li>python312Packages.llm-perplexity.dist</li>
    <li>python312Packages.llm-templates-fabric</li>
    <li>python312Packages.llm-templates-fabric.dist</li>
    <li>python312Packages.llm-templates-github</li>
    <li>python312Packages.llm-templates-github.dist</li>
    <li>python312Packages.llm-tools-datasette</li>
    <li>python312Packages.llm-tools-datasette.dist</li>
    <li>python312Packages.llm-tools-quickjs</li>
    <li>python312Packages.llm-tools-quickjs.dist</li>
    <li>python312Packages.llm-tools-simpleeval</li>
    <li>python312Packages.llm-tools-simpleeval.dist</li>
    <li>python312Packages.llm-tools-sqlite</li>
    <li>python312Packages.llm-tools-sqlite.dist</li>
    <li>python312Packages.llm-venice</li>
    <li>python312Packages.llm-venice.dist</li>
    <li>python312Packages.llm-video-frames</li>
    <li>python312Packages.llm-video-frames.dist</li>
    <li>python312Packages.llm.dist</li>
    <li>python312Packages.llmx</li>
    <li>python312Packages.llmx.dist</li>
    <li>python312Packages.markitdown</li>
    <li>python312Packages.markitdown.dist</li>
    <li>python312Packages.openai</li>
    <li>python312Packages.openai-agents</li>
    <li>python312Packages.openai-agents.dist</li>
    <li>python312Packages.openai.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.pypasser</li>
    <li>python312Packages.pypasser.dist</li>
    <li>python312Packages.speechrecognition</li>
    <li>python312Packages.speechrecognition.dist</li>
    <li>python312Packages.sqlite-vec</li>
    <li>python312Packages.sqlite-vec.dist</li>
    <li>python312Packages.tami4edgeapi</li>
    <li>python312Packages.tami4edgeapi.dist</li>
    <li>python313Packages.banks</li>
    <li>python313Packages.banks.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.fireworks-ai</li>
    <li>python313Packages.fireworks-ai.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.litellm</li>
    <li>python313Packages.litellm.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.llm</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.llm-cmd</li>
    <li>python313Packages.llm-cmd.dist</li>
    <li>python313Packages.llm-command-r</li>
    <li>python313Packages.llm-command-r.dist</li>
    <li>python313Packages.llm-deepseek</li>
    <li>python313Packages.llm-deepseek.dist</li>
    <li>python313Packages.llm-docs</li>
    <li>python313Packages.llm-docs.dist</li>
    <li>python313Packages.llm-echo</li>
    <li>python313Packages.llm-echo.dist</li>
    <li>python313Packages.llm-fragments-github</li>
    <li>python313Packages.llm-fragments-github.dist</li>
    <li>python313Packages.llm-fragments-pypi</li>
    <li>python313Packages.llm-fragments-pypi.dist</li>
    <li>python313Packages.llm-fragments-reader</li>
    <li>python313Packages.llm-fragments-reader.dist</li>
    <li>python313Packages.llm-fragments-symbex</li>
    <li>python313Packages.llm-fragments-symbex.dist</li>
    <li>python313Packages.llm-gemini</li>
    <li>python313Packages.llm-gemini.dist</li>
    <li>python313Packages.llm-gguf</li>
    <li>python313Packages.llm-gguf.dist</li>
    <li>python313Packages.llm-git</li>
    <li>python313Packages.llm-git.dist</li>
    <li>python313Packages.llm-github-copilot</li>
    <li>python313Packages.llm-github-copilot.dist</li>
    <li>python313Packages.llm-grok</li>
    <li>python313Packages.llm-grok.dist</li>
    <li>python313Packages.llm-groq</li>
    <li>python313Packages.llm-groq.dist</li>
    <li>python313Packages.llm-hacker-news</li>
    <li>python313Packages.llm-hacker-news.dist</li>
    <li>python313Packages.llm-jq</li>
    <li>python313Packages.llm-jq.dist</li>
    <li>python313Packages.llm-llama-server</li>
    <li>python313Packages.llm-llama-server.dist</li>
    <li>python313Packages.llm-mistral</li>
    <li>python313Packages.llm-mistral.dist</li>
    <li>python313Packages.llm-ollama</li>
    <li>python313Packages.llm-ollama.dist</li>
    <li>python313Packages.llm-openai-plugin</li>
    <li>python313Packages.llm-openai-plugin.dist</li>
    <li>python313Packages.llm-openrouter</li>
    <li>python313Packages.llm-openrouter.dist</li>
    <li>python313Packages.llm-pdf-to-images</li>
    <li>python313Packages.llm-pdf-to-images.dist</li>
    <li>python313Packages.llm-perplexity</li>
    <li>python313Packages.llm-perplexity.dist</li>
    <li>python313Packages.llm-templates-fabric</li>
    <li>python313Packages.llm-templates-fabric.dist</li>
    <li>python313Packages.llm-templates-github</li>
    <li>python313Packages.llm-templates-github.dist</li>
    <li>python313Packages.llm-tools-datasette</li>
    <li>python313Packages.llm-tools-datasette.dist</li>
    <li>python313Packages.llm-tools-quickjs</li>
    <li>python313Packages.llm-tools-quickjs.dist</li>
    <li>python313Packages.llm-tools-simpleeval</li>
    <li>python313Packages.llm-tools-simpleeval.dist</li>
    <li>python313Packages.llm-tools-sqlite</li>
    <li>python313Packages.llm-tools-sqlite.dist</li>
    <li>python313Packages.llm-venice</li>
    <li>python313Packages.llm-venice.dist</li>
    <li>python313Packages.llm-video-frames</li>
    <li>python313Packages.llm-video-frames.dist</li>
    <li>python313Packages.llm.dist</li>
    <li>python313Packages.llmx</li>
    <li>python313Packages.llmx.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.openai-agents</li>
    <li>python313Packages.openai-agents.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.pypasser</li>
    <li>python313Packages.pypasser.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.sqlite-vec</li>
    <li>python313Packages.sqlite-vec.dist</li>
    <li>python313Packages.tami4edgeapi</li>
    <li>python313Packages.tami4edgeapi.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>sploitscan</li>
    <li>sploitscan.dist</li>
    <li>textlsp</li>
    <li>textlsp.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.fnllm</summary>
<pre>
fnllm_tests/unit/llm/services/test_json.py:27
  /private/tmp/nix-build-python3.12-fnllm-0.2.8.drv-0/fnllm-0.2.8/fnllm_tests/unit/llm/services/test_json.py:27: PytestCollectionWarning: cannot collect test class 'TestLLM' because it has a __init__ constructor (from: fnllm_tests/unit/llm/services/test_json.py)
    class TestLLM(BaseLLM):

fnllm_tests/unit/llm/services/test_retryer.py:15
  /private/tmp/nix-build-python3.12-fnllm-0.2.8.drv-0/fnllm-0.2.8/fnllm_tests/unit/llm/services/test_retryer.py:15: PytestCollectionWarning: cannot collect test class 'TestError' because it has a __init__ constructor (from: fnllm_tests/unit/llm/services/test_retryer.py)
    class TestError(BaseException): ...

../../../../nix/store/zgxgy76v7z022c80jmzk6klchnc4ky9c-python3.12-polyfactory-2.22.1/lib/python3.12/site-packages/polyfactory/factories/base.py:246
  /nix/store/zgxgy76v7z022c80jmzk6klchnc4ky9c-python3.12-polyfactory-2.22.1/lib/python3.12/site-packages/polyfactory/factories/base.py:246: DeprecationWarning: Use of deprecated default '__check_model__'. Deprecated in polyfactory v2.22.0. This default will be removed in the next major version. Use 'set to `False` explicitly to keep existing behaviour' instead
    warn_deprecation(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED fnllm_tests/unit/openai/llm/features/test_tool_parsing.py::test_tools_are_parsed - TypeError: Cannot instantiate typing.Union
FAILED fnllm_tests/unit/openai/llm/features/test_tool_parsing.py::test_invalid_tool_arguments_should_raise - TypeError: Cannot instantiate typing.Union
FAILED fnllm_tests/unit/openai/llm/features/test_tool_parsing.py::test_tool_not_found_should_raise - TypeError: Cannot instantiate typing.Union
FAILED fnllm_tests/unit/openai/llm/test_utils.py::test_tool_calls_to_params - TypeError: Cannot instantiate typing.Union
=========== 4 failed, 75 passed, 58 deselected, 4 warnings in 7.24s ============</pre>
</details>
<details>
<summary>python312Packages.instructor</summary>
<pre>
    def __call__(self, *args, **kwds):
>       raise TypeError(f"Cannot instantiate {self!r}")
E       TypeError: Cannot instantiate typing.Union

/nix/store/dm4fl983hky9lxj9bw3l7i0jnb4j408m-python3-3.12.11/lib/python3.12/typing.py:501: TypeError
=============================== warnings summary ===============================
tests/test_function_calls.py::test_complete_output_no_exception
  /private/tmp/nix-build-python3.12-instructor-1.7.9.drv-0/source/instructor/function_calls.py:243: DeprecationWarning: The FUNCTIONS mode is deprecated and will be removed in future versions
    Mode.warn_mode_functions_deprecation()

tests/test_function_calls.py::test_incomplete_output_exception_raise[mock_completion0]
  tests/test_function_calls.py:140: PytestWarning: The test <Function test_incomplete_output_exception_raise[mock_completion0]> is marked with '@pytest.mark.asyncio' but it is not an async function. Please remove the asyncio mark. If the test is not marked explicitly, check for global marks applied via 'pytestmark'.
    @pytest.mark.asyncio  # type: ignore[misc]

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_function_calls.py::test_no_refusal_attribute - TypeError: Cannot instantiate typing.Union
FAILED tests/test_function_calls.py::test_missing_refusal_attribute - TypeError: Cannot instantiate typing.Union
===== 2 failed, 184 passed, 4 skipped, 18 deselected, 2 warnings in 2.01s ======</pre>
</details>
<details>
<summary>python313Packages.fnllm</summary>
<pre>
fnllm_tests/unit/llm/services/test_json.py:27
  /private/tmp/nix-build-python3.13-fnllm-0.2.8.drv-0/fnllm-0.2.8/fnllm_tests/unit/llm/services/test_json.py:27: PytestCollectionWarning: cannot collect test class 'TestLLM' because it has a __init__ constructor (from: fnllm_tests/unit/llm/services/test_json.py)
    class TestLLM(BaseLLM):

fnllm_tests/unit/llm/services/test_retryer.py:15
  /private/tmp/nix-build-python3.13-fnllm-0.2.8.drv-0/fnllm-0.2.8/fnllm_tests/unit/llm/services/test_retryer.py:15: PytestCollectionWarning: cannot collect test class 'TestError' because it has a __init__ constructor (from: fnllm_tests/unit/llm/services/test_retryer.py)
    class TestError(BaseException): ...

../../../../nix/store/8iwnnrxgzxd8v8cnw7cw8jwl5k9cqbds-python3.13-polyfactory-2.22.1/lib/python3.13/site-packages/polyfactory/factories/base.py:246
  /nix/store/8iwnnrxgzxd8v8cnw7cw8jwl5k9cqbds-python3.13-polyfactory-2.22.1/lib/python3.13/site-packages/polyfactory/factories/base.py:246: DeprecationWarning: Use of deprecated default '__check_model__'. Deprecated in polyfactory v2.22.0. This default will be removed in the next major version. Use 'set to `False` explicitly to keep existing behaviour' instead
    warn_deprecation(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED fnllm_tests/unit/openai/llm/features/test_tool_parsing.py::test_tools_are_parsed - TypeError: Cannot instantiate typing.Union
FAILED fnllm_tests/unit/openai/llm/features/test_tool_parsing.py::test_invalid_tool_arguments_should_raise - TypeError: Cannot instantiate typing.Union
FAILED fnllm_tests/unit/openai/llm/features/test_tool_parsing.py::test_tool_not_found_should_raise - TypeError: Cannot instantiate typing.Union
FAILED fnllm_tests/unit/openai/llm/test_utils.py::test_tool_calls_to_params - TypeError: Cannot instantiate typing.Union
=========== 4 failed, 75 passed, 58 deselected, 4 warnings in 7.52s ============</pre>
</details>
<details>
<summary>python313Packages.instructor</summary>
<pre>
    def __call__(self, *args, **kwds):
>       raise TypeError(f"Cannot instantiate {self!r}")
E       TypeError: Cannot instantiate typing.Union

/nix/store/v72ywdz0dd2d6sb3knymx9ww8r6pcqba-python3-3.13.5/lib/python3.13/typing.py:560: TypeError
=============================== warnings summary ===============================
tests/test_function_calls.py::test_complete_output_no_exception
  /private/tmp/nix-build-python3.13-instructor-1.7.9.drv-0/source/instructor/function_calls.py:243: DeprecationWarning: The FUNCTIONS mode is deprecated and will be removed in future versions
    Mode.warn_mode_functions_deprecation()

tests/test_function_calls.py::test_incomplete_output_exception_raise[mock_completion0]
  tests/test_function_calls.py:140: PytestWarning: The test <Function test_incomplete_output_exception_raise[mock_completion0]> is marked with '@pytest.mark.asyncio' but it is not an async function. Please remove the asyncio mark. If the test is not marked explicitly, check for global marks applied via 'pytestmark'.
    @pytest.mark.asyncio  # type: ignore[misc]

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_function_calls.py::test_no_refusal_attribute - TypeError: Cannot instantiate typing.Union
FAILED tests/test_function_calls.py::test_missing_refusal_attribute - TypeError: Cannot instantiate typing.Union
===== 2 failed, 184 passed, 4 skipped, 18 deselected, 2 warnings in 2.05s ======</pre>
</details>
</details>

--- a/pkgs/development/python-modules/openai/default.nix
+++ b/pkgs/development/python-modules/openai/default.nix
@@ -2,7 +2,6 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-  pythonAtLeast,
 
   # build-system
   hatchling,
@@ -18,10 +17,9 @@
   tqdm,
   typing-extensions,
 
-  # `httpx_aiohttp` not currently in `nixpkgs`
   # optional-dependencies (aiohttp)
-  # aiohttp,
-  # httpx_aiohttp,
+  aiohttp,
+  httpx-aiohttp,
 
   # optional-dependencies (datalib)
   numpy,
@@ -45,20 +43,22 @@
   respx,
 
   # optional-dependencies toggle
+  withAiohttp ? true,
+  withDatalib ? false,
   withRealtime ? true,
   withVoiceHelpers ? true,
 }:
 
 buildPythonPackage rec {
   pname = "openai";
-  version = "1.99.1";
+  version = "1.99.9";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "openai";
     repo = "openai-python";
     tag = "v${version}";
-    hash = "sha256-TFzLfCT71BSbKg7LSqpFAsutKYAeQ6ALy7AE4ldeHr8=";
+    hash = "sha256-gE74uUQTj2kL9kwsRdu4IW69BGRmEOCRjDRiZCN8TEA=";
   };
 
   postPatch = ''substituteInPlace pyproject.toml --replace-fail "hatchling==1.26.3" "hatchling"'';
@@ -78,15 +78,16 @@ buildPythonPackage rec {
     tqdm
     typing-extensions
   ]
+  ++ lib.optionals withAiohttp optional-dependencies.aiohttp
+  ++ lib.optionals withDatalib optional-dependencies.datalib
   ++ lib.optionals withRealtime optional-dependencies.realtime
   ++ lib.optionals withVoiceHelpers optional-dependencies.voice-helpers;
 
   optional-dependencies = {
-    # `httpx_aiohttp` not currently in `nixpkgs`
-    # aiohttp = [
-    #   aiohttp
-    #   httpx_aiohttp
-    # ];
+    aiohttp = [
+      aiohttp
+      httpx-aiohttp
+    ];
     datalib = [
       numpy
       pandas
@@ -114,23 +115,12 @@ buildPythonPackage rec {
     respx
   ];
 
-  pytestFlags = [
-    "-Wignore::DeprecationWarning"
-  ];
-
-  disabledTests = [
-    # Tests make network requests
-    "test_copy_build_request"
-    "test_basic_attribute_access_works"
-  ]
-  ++ lib.optionals (pythonAtLeast "3.13") [
-    # RuntimeWarning: coroutine method 'aclose' of 'AsyncStream._iter_events' was never awaited
-    "test_multi_byte_character_multiple_chunks"
-  ];
-
   disabledTestPaths = [
     # Test makes network requests
     "tests/api_resources"
+    # E   TypeError: Unexpected type for 'content', <class 'inline_snapshot._external.external'>
+    # This seems to be due to `inline-snapshot` being disabled when `pytest-xdist` is used.
+    "tests/lib/chat/test_completions_streaming.py"
   ];
 
   meta = {

